### PR TITLE
Multimap test cases

### PR DESCRIPTION
**Pull request by utku-caglayan**

### CODE REVIEW

File: multi_map_it_test.go [hunk 1]
```diff
@@ -450,3 +450,120 @@ func TestMultiMap_TryLockWithLeaseAndTimeout(t *testing.T) {
 		assert.False(t, b)
 	})
 }
+
+func TestMultiMap_ContainsKey(t *testing.T) {
+	it.MultiMapTester(t, func(t *testing.T, m *hz.MultiMap) {
+		ctx := context.Background()
+		keyExists, err := m.ContainsKey(ctx, "key")
```
Comment (yuce) [inline multi_map_it_test.go:457]: How about `exists` instead of `keyExists`?

File: multi_map_it_test.go [hunk 2]
```diff
@@ -450,3 +450,120 @@ func TestMultiMap_TryLockWithLeaseAndTimeout(t *testing.T) {
...
+			t.Fatal(err)
+		}
+		assert.True(t, keyExists)
+	})
+}
+
+func TestMultiMap_ContainsValue(t *testing.T) {
+	it.MultiMapTester(t, func(t *testing.T, m *hz.MultiMap) {
+		ctx := context.Background()
+		valueExists, err := m.ContainsValue(ctx, "value")
```
Comment (yuce) [inline multi_map_it_test.go:475]: How about `exists` instead of valueExists?

File: multi_map_it_test.go [hunk 3]
```diff
@@ -450,3 +450,120 @@ func TestMultiMap_TryLockWithLeaseAndTimeout(t *testing.T) {
...
+				triggerEvent: func() {
+					it.MustBool(m.Put(ctx, key, "testValue"))
+					err := m.Clear(ctx)
+					if err != nil {
+						t.Fatal(err)
+					}
+				},
+			},
+		}
+		opSucceed := false
```
Comment (yuce) [inline multi_map_it_test.go:544]: how about `success` instead of `opSucceed`?

**Review by yuce — APPROVED** [review]
Thanks, Looks good!

File: multi_map_it_test.go (hunk 1 above)
Comment (utku-caglayan) [reply multi_map_it_test.go]: Thanks, fixed it at [`c1204cb`](https://github.com/hazelcast/hazelcast-go-client/pull/697/commits/c1204cbf69648056914f16f60ce67946a92543bd)

File: multi_map_it_test.go (hunk 2 above)
Comment (utku-caglayan) [reply multi_map_it_test.go]: Thanks, fixed it at c1204cb

File: multi_map_it_test.go (hunk 3 above)
Comment (utku-caglayan) [reply multi_map_it_test.go]: Thanks, fixed it at c1204cb

**Review by emreyigit — APPROVED** [review]
Looks great!

**Pull request merged**